CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The order and collapsed state of columns is now persisted across page navigation
 
 ### Bug fixes and improvements
 
+- server: reusing Postgres scalars in custom types is allowed (close #4125)
```

Comment:
This is probably subjective, but I think this would read more positively if written in [the active voice](https://en.wikipedia.org/wiki/Active_voice)! Something like this:

```suggestion
- server: Support reusing Postgres scalars in custom types (close #4125)
```